### PR TITLE
bbn-1: FP registry guide

### PR DESCRIPTION
**Pull request by RafilxTenfen**
Registration guide for bbn-1

TODO:
- [x] Update link to terms and conditions

### CODE REVIEW

File: bbn-1/finality-providers/README.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+- `website`: optional website link URL.
+- `security_contact`: required email for security contact.
+- `details`: any other optional detail information.
+- `eots_pk`: the btc pub key in hex format (The `pub_key_hex` property of the
+`eotsd keys add` command output).
+- `commission`: the commission charged for BTC staking rewards.
+The commission will be parsed as a decimal:
+  - `"1.00"` represents 100% commission.
+  - `"0.10"` represents  10% commission.
+  - `"0.01"` represents  01% commission.
```
Comment (vitsalis) [inline bbn-1/finality-providers/README.md:262]: Let's make this example the minimum commission rate.

File: bbn-1/finality-providers/README.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+
+**⚠ Warning!**
+The minimum commission value accepted is 5%. It will remain immutable and can't
```
Comment (vitsalis) [inline bbn-1/finality-providers/README.md:265]: Let's wait until the decision for this is finalized.

**Review by vitsalis — APPROVED** [review]
lgtm, some small comments

File: bbn-1/finality-providers/README.md (hunk 2 above)
Comment (RafilxTenfen) [reply bbn-1/finality-providers/README.md:265]: 3%, I will update

File: .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+instructions to generate information.
+
+## Checklist
+
+- [ ] [Create EOTS Key](https://github.com/babylonlabs-io/networks/blob/ef18868512b0b9c823c653cdabc975f88b6fc7a2/bbn-1/finality-providers/README.md#L154)
+- [ ] Backup Mnemonic
+- [ ] Generate Finality Provider Information
+- [ ] Input the information into a file under `bbn-1/finality-providers/registry/{nickname}.json`
+- [ ] [Sign the file](https://github.com/babylonlabs-io/networks/blob/ef18868512b0b9c823c653cdabc975f88b6fc7a2/bbn-1/finality-providers/README.md#L271)
+- [ ] Input the signature into a file under `bbn-1/finality-providers/sigs/{nickname}.sig`
```
[the remark this answers is not in the document]
Comment (RafilxTenfen) [reply .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md]: https://github.com/babylonlabs-io/networks/pull/1/commits/502fc3f824d3a163448dcdcea3fe89e29989b144

File: bbn-1/finality-providers/README.md (hunk 1 above)
Comment (RafilxTenfen) [reply bbn-1/finality-providers/README.md]: https://github.com/babylonlabs-io/networks/pull/1/commits/502fc3f824d3a163448dcdcea3fe89e29989b144

File: bbn-1/finality-providers/README.md (hunk 2 above)
Comment (RafilxTenfen) [reply bbn-1/finality-providers/README.md]: https://github.com/babylonlabs-io/networks/pull/1/commits/502fc3f824d3a163448dcdcea3fe89e29989b144

**Comment by m3diumrare** [comment]
I see people are already submitting FP registration PRs but the guide here is not merged yet. 

Will anything possibly change that would invalidate existing submissions, or should I add a PR now too?

**Comment by RafilxTenfen** [comment]
> I see people are already submitting FP registration PRs but the guide here is not merged yet.
> 
> Will anything possibly change that would invalidate existing submissions, or should I add a PR now too?

You should not create the PR, I will need to ask for everyone to rebase and update the PR's once the link for terms and condition is finish 
@m3diumrare

**Comment by Puckapao** [comment]
> > I see people are already submitting FP registration PRs but the guide here is not merged yet.
> > Will anything possibly change that would invalidate existing submissions, or should I add a PR now too?
> 
> You should not create the PR, I will need to ask for everyone to rebase and update the PR's once the link for terms and condition is finish @m3diumrare

Hello Raflix,
May I ask?
So now we can submit right? I saw the deadline has been added but still no terms and conditions, also the email has noted that I should PR before the deadline so I did it right away and got KYB url from babylon.synaps.me sent to me in email to do KYB next (which I already did it).

**Comment by RafilxTenfen** [comment]
> Hello Raflix, May I ask? So now we can submit right? I saw the deadline has been added but still no terms and conditions, also the email has noted that I should PR before the deadline so I did it right away and got KYB url from babylon.synaps.me sent to me in email to do KYB next (which I already did it).

Hey @Puckapao, you should not create the PR while this PR #1 is not merged

File: .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# New ${nickname} Finality Provider
+
+## Checklist
+
+- [ ] Follow the finality provider information registry
```
Comment (vitsalis) [inline .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md:5]:
```suggestion
- [ ] I have followed the finality provider information registry
```

File: .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+[guide](https://github.com/babylonlabs-io/networks/blob/ef18868512b0b9c823c653cdabc975f88b6fc7a2/bbn-1/finality-providers/README.md#L1)
+- [ ] Backup Mnemonic
```
Comment (vitsalis) [inline .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md:7]:
```suggestion
- [ ] I have backed up my mnemonic
```

File: .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+- [ ] I have read and agree to the [Babylon Ecosystem Participant License](https://docs.babylonlabs.io/assets/files/babylon-ecosystem-participant-license.pdf) and the [Babylon Ecosystem Participant Agreement](https://docs.babylonlabs.io/assets/files/babylon-ecosystem-participant-agreement.pdf).
+
+> [!CAUTION]
+> The loss of the (generated keys + mnemonic) makes the finality provider
+useless and unable to provide finality, which would lead to no transition to
+later phases of the Babylon networks.
```
Comment (vitsalis) [inline .github/PULL_REQUEST_TEMPLATE/new_finality_provider.md:13]:
```suggestion
later phases of the Babylon network.
```

File: bbn-1/finality-providers/README.md [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+For security and quality reasons, strict eligibility criteria are imposed,
+focusing on the finality provider's proven commitment and identity
+verification. More specifically, for a finality provider to be eligible for
+inclusion in this information registry, it needs to meet the following
+requirements:
+
+- having participated in the
+[Babylon testnet-4](https://github.com/babylonchain/networks/tree/ac531139d5a75e575b34a80c9f8fc841cc33adab/bbn-test-4),
+- to go through a know your business (KYB) process conducted by Babylon Labs, and
+- to submit a pull request (PR), and the deadline is **August 16th, end of the
```
Comment (vitsalis) [inline bbn-1/finality-providers/README.md:42]:
```suggestion
- to submit a pull request (PR) before the deadline of **August 16th, end of the
```

**Review by vitsalis — APPROVED** [review]

**Review by vitsalis — APPROVED** [review]

**Pull request merged**